script/hassfest/quality_scale_validation/runtime_data.py

```diff
@@ -10,7 +10,7 @@
 from script.hassfest import ast_parse_module
 from script.hassfest.model import Config, Integration
 
-_ANNOTATION_MATCH = re.compile(r"^[A-Za-z]+ConfigEntry$")
+_ANNOTATION_MATCH = re.compile(r"^[A-Za-z0-9]+ConfigEntry$")
```

Comment:
Python classes cannot start with a number

```suggestion
_ANNOTATION_MATCH = re.compile(r"^[A-Za-z][A-Za-z0-9]+ConfigEntry$")
```